### ISSUE TITLE
Change text string "Install Split APKs"

### ISSUE THREAD

**Issue by Valdnet** [SmartPack/PackageManager#51]
The text string "Install Split APKs" is somewhat confusing to the user. If the user wants to install the APK installer, he doesn't know what to choose from the menu because the text string points to "Split APKs".

https://github.com/SmartPack/PackageManager/blob/7b0bbc712adf734f0c0ef97db9d3d8547cdffa64/app/src/main/res/values/strings.xml#L79

Apps:
Android 10
Package Manager v5.2

**Comment by sunilpaulmathew** [comment]
> The text string "Install Split APKs" is somewhat confusing to the user. If the user wants to install the APK installer, he doesn't know what to choose from the menu because the text string points to "Split APKs".

Are you suggesting to add separate menu for APKs/Split APKs/App Bundles?
An alternate way is to show a detailed information screen on the first launch of that feature. Also, we may simplify the title into "**Installer**" or something similar.

**Comment by Valdnet** [comment]
> Also, we may simplify the title into "Installer" or something similar.

Very good idea 👍.

**Comment by sunilpaulmathew** [comment]
> > Also, we may simplify the title into "Installer" or something similar.
> 
> Very good idea 👍.
> 
> @Aga-C Although she will have something to translate again 😁.

This is something in my mind. Suggestions are most welcome.
![Screenshot_1631480682](https://user-images.githubusercontent.com/15515387/133002791-0f3d0bbb-ad90-4a7f-a1f9-a120ea973987.png)

**Comment by Aga-C** [comment]
@sunilpaulmathew For me, it looks great 👍 But I noticed that it is not possible to install an individual APK file, only split APKs and bundles. For APKs, it always takes the whole directory.

Also, when you add new strings for translation, it would be nice if you wait a while before releasing a new version 🙂

**Comment by Valdnet** [comment]
> For me, it looks great

It turns out that there is something wrong with the icons of the installation files only for me.

> Also, when you add new strings for translation, it would be nice if you wait a while before releasing a new version

I also had a problem with this with developers, but we sorted it out somehow.

**Comment by sunilpaulmathew** [comment]
> Also, when you add new strings for translation, it would be nice if you wait a while before releasing a new version 🙂

Will be done.

**Comment by Valdnet** [comment]
@sunilpaulmathew Looks very good. We will see how it will look in general in the finished application and in functionality for the user 😉.

**Comment by sunilpaulmathew** [comment]
> @sunilpaulmathew Looks very good. We will see how it will look in general in the finished application and in functionality for the user 😉.

I'll create a test build hopefully today evening. Also, I'll try to make it official with in next two days so that I can go for a vacation :)
Btw, it seems like we are living in nearby?

**Comment by Valdnet** [comment]
> I'll create a test build hopefully today evening.

Cool. I will have something to test 😉.

> Btw, it seems like we are living in nearby?

This is interesting. You live in Poland?

**Comment by sunilpaulmathew** [comment]
> This is interesting. You live in Poland?

Nope, but in Czech.

**Comment by Valdnet** [comment]
You live close to Poland 😉. I thought you live in India.

**Comment by sunilpaulmathew** [comment]
Btw, if you guys are active in Telegram, feel free to join (if you haven't already) our telegram group at https://t.me/smartpack_kmanager

**Comment by sunilpaulmathew** [comment]
@Valdnet @Aga-C 
So, [here](https://www.dropbox.com/s/rvi05lbi7yeya2i/com.smartpack.packagemanager.apk?dl=0) is a test build, if you like to test the new changes. 

Btw, before you wonder, I completely removed using external file picker from the app, as it has become a big headache later. I hope the in-built file picker is enough and fine (of course, I'll try to implement the changes suggested by @Valdnet).

**Comment by Valdnet** [comment]
> I completely removed using external file picker from the app

@sunilpaulmathew Very good in my opinion 👍. The in-built file picker is sufficient for this application.

I start testing your application as soon as I finish testing another one 😉.

**Comment by Valdnet** [comment]
Notes for the test application version:

1. I think that if there is no other file manager to choose from, the message "Please Note" is unnecessary.

2. I would add an option in the application settings "Reset Messages" with the description "Again enables messages that have been disabled.". Because you don't always remember what it was about or you turn it off too quickly without reading it. This option will allow you to display them again.
3. I would change the text string "Never shows this page again!" to "Do not show this information".

**Comment by sunilpaulmathew** [comment]
> 1. I think that if there is no other file manager to choose from, the message "Please Note" is unnecessary.

Is it still there? I thought I removed it already.

> 2. I would add an option in the application settings "Reset messages" with the description "Again enables messages that have been disabled.". Because you don't always remember what it was about or you turn it off too quickly without reading it. This option will allow you to display them again.

Let me see how to handle it.

> 3. I would change the text string "Never shows this page again!" to "Do not show this information".

Will be done.

Thanks a lot.

**Comment by Aga-C** [comment]
>Is it still there? I thought I removed it already.

No, it's not there now.

**Comment by Valdnet** [comment]
> No, it's not there now.

You're right. I turned off the message, and then I forgot if there was still this information. I made a description of issue from the screenshot above.
This confirms the problem with the exclusion of information that I talked about in point 2 :smile:.

**Comment by sunilpaulmathew** [comment]
Okay guys, I'll work on these things as much as possible tonight and upload another test build + strings in POEditor for translation. I would like to release a new version tomorrow night or Thursday morning (then, I'll be on vacation for a week).

**Comment by Valdnet** [comment]
For me, the function works fine 👍.

**Comment by Valdnet** [comment]
I propose to change:

1. Normal font.
2. Remove the exclamation point in the text string `Do not show this again!`.

![obraz](https://user-images.githubusercontent.com/47037905/133500459-e0cfaa58-e351-4527-aeed-a25734ef8565.png)
